packages/commonwealth/server/socket/chainEventsNs.ts

```diff
@@ -1,75 +0,0 @@
-import { Server } from 'socket.io';
```

Comment:
Checked that these socket files are safe to delete?

Reply:
@timolegros think this was your stuff